sklearn/metrics/classification.py

```diff
@@ -1716,7 +1717,8 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
     y_true : array-like or label indicator matrix
         Ground truth (correct) labels for n_samples samples.
 
-    y_pred : array-like of float, shape = (n_samples, n_classes) or (n_samples,)
+    y_pred : array-like of float, 
```

Comment:
actually can you do a line break with ``\`` here? The shape should be on the same line in the rendered docs.

Reply:
Yep, sorry, done!